### DISCUSSION TITLE
Suppress CVE with expiration date for false positive CVE's.

### DISCUSSION THREAD

**Discussion by Swapnil-CSI** [DependencyTrack/dependency-track#2281] [General]
Is there any way to suppress the CVE's with expiration date. So after the expiration date CVE will show again in scanning report.
Similar functionality is available under dependency check using until option.

```
    <suppress until="2022-11-14Z">
        <notes><![CDATA[
            temporary suppress test package 
        ]]></notes>
        <sha1>43ddxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx51f</sha1>
        <cve>CVE-2022-XXXX</cve>
    </suppress>
```

**Comment by SaurabhAhuja1983** [comment]
@Swapnil-CSI did you find the solution to this ?

**Reply by Swapnil-CSI** [reply]
@SaurabhAhuja1983 Nope.
